account_bank_statement_import_camt_oca/models/parser.py

```diff
@@ -97,12 +97,16 @@ def parse_transaction_details(self, ns, node, transaction):
         )
         if party_node:
             name_node = node.xpath(
-                "./ns:RltdPties/ns:%s/ns:Nm" % party_type, namespaces={"ns": ns}
+                "./ns:RltdPties/ns:{pt}/ns:Nm | ./ns:RltdPties/ns:{pt}/ns:Pty/ns:Nm".format(
```

Comment:
Why not use f-string instead of format(). The former will fit in our 120 chars limit and will be more readable IMHO.